modules/front-end/src/app/core/components/change-review/change-review.component.html

```diff
@@ -29,7 +29,7 @@
         <form nz-form [formGroup]="reviewForm" nzLayout="vertical">
           <nz-form-item>
             <nz-form-label nzRequired class="section-title" i18n="@@common.comment">Comment</nz-form-label>
-            <nz-form-control i18n-nzErrorTip="@@common.comment-cannot-be-empty" nzErrorTip="Comment cannot be empty">
```

Comment:
Good start point, this removes the error message. Some more code need to change:

- Current file Line 31: remove **nzRequired**, this would removes the asterisk (*)
- modules/front-end/src/app/core/components/change-review/change-review.component.ts Line 38: do the following change: `comment: ['', [Validators.required]]` => `comment: ['', []]`

Reply:
Thank you, I have modified it. Please have a look. @cosmos-explorer